### DISCUSSION TITLE
Welcome to pf2e-wiki Discussions!

### DISCUSSION THREAD

**Discussion by reyzor1991** [reyzor1991/pf2e-wiki#56] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by slohcin13** [comment]
Hey, I'm running a few PF2e campaigns for some friend groups and tripped across your PF2e Automations mod while trying to find ways to improve the flow of spells/actions/reactions triggering effects.  I have had some luck creating a few automation rules but was wondering if there is anywhere the various trigger types are explained? I see quite a few are listed in the wiki, but I'm not sure what the purpose of most of them is.
